broker/src/test/java/io/zeebe/broker/system/configuration/BrokerCfgTest.java

```diff
@@ -58,7 +51,7 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-public final class ConfigurationTest {
+public final class BrokerCfgTest {
```

Comment:
Nit: as this is testing configuration parsing and not just the `BrokerCfg` class, I'd leave it as `ConfigurationTest`.